File: SOLUTIONS/findASCIIvalue.java
import java.util.Scanner;

public class findASCIIvalue{
	public static void main(String[] args){
		Scanner input = new Scanner(System.in);
		System.out.println("enter string");
		String str= input.next();
		char[] stringToCharArray = str.toCharArray();
			int i,j = 0;
			int ascii= 1;
			for(i=0;i<stringToCharArray.length;i++)
			{
				 j=(int)stringToCharArray[i];
				 ascii = ascii*j;
				 
			}
			System.out.println("product of ascii values="+ascii);

	}
}

File: SOLUTIONS/findASCIIvalue.dart
import 'dart:io';

void main() {
  print("enter string");
  String str = stdin.readLineSync(encoding: Encoding.UTF_8);
  int i, j = 0;
  int ascii = 1;
  for (i = 0; i < str.length; i++) {
    j = int.parse(str[i]);
    ascii = ascii * j;
  }
  print("product of ascii values=" + ascii);
}
